src/client/ReservationForms.js

```diff
@@ -0,0 +1,95 @@
+import React, { useState } from 'react'
+import { Link } from "react-router-dom";
+
+import 'bootstrap/dist/css/bootstrap.min.css';
+import Form from 'react-bootstrap/Form';
+import Button from 'react-bootstrap/Button';
+
+import './MealsWithID.css'
+
+function showCurrentDate() {
+    const today = new Date();
```

Comment:
I suggest using a library like moment.js to handle date. There is no need for us to write this logic ourselves

Reply:
Thanks a lot, Niels, I'll work on this now.